source/administration/config-in-database.rst

```diff
@@ -10,24 +10,175 @@ Benefits to using this option:
   - Automatically deploy SAML certificates and keys to all servers in the cluster.
 
-To start using configuration in database, pass the database connection string via the ``--config`` flag or ``MM_CONFIG`` environment variable. For example:
+How to Migrate Configuration to the Database
+--------------------------------------------
 
-  .. code-block:: text
-  
-    bin/mattermost --config="postgres://mmuser:mostest@localhost:5432/mattermost_test?sslmode=disable\u0026connect_timeout=10"
+These are complete instructions for migrating the Mattermost configuration to the database, as well as updating your ``systemd`` configuration to load the configuration from the database.
 
-To migrate an existing config.json into the database, use the ``config migrate`` `command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-config-migrate>`_. For example:
+**NOTE:** These instructions assume you have Mattermost server installed at ``/opt/mattermost`` If you are running Mattermost in a different directory you will have to modify the paths to match your environment.
 
-  .. code-block:: text
+1. Get your Database Connection String
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    bin/mattermost config migrate  path/to/config.json "postgres://mmuser:mostest@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10"
+The first step is to get your master database connection string. There are several ways to do this, but the easiest is touse ``mattermost config get`` command like this:
 
-Any existing SAML certificates and private keys will also be migrated to the database.
+.. code-block:: bash
 
+   sudo su mattermost
+   cd /opt/mattermost
+   bin/mattermost config get SqlSettings.DataSource
+
+Output:
+
+.. code-block:: text
+
+   SqlSettings.DataSource: "mmuser:really_secure_password@tcp(127.0.0.1:3306)/mattermost?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s"
+
+**NOTE:** Be sure to run this command as the ``mattermost`` user and *not* root. Running the Mattermost binary as root will cause permissions errors.
+
+Another way is to view your ``config.json`` file and get the value in ``SqlSettings.DataSource``. 
+
+If ``SqlSettings.DataSource`` does not start with ``postgres://`` then you have to add ``mysql://`` to the beginning. Also, if you see ``\u0026`` replace it with ``&``.
+
+Here are two example connection strings:
+
+*MySQL*
+
+.. code-block:: text
+
+   mysql://mmuser:really_secure_password@tcp(127.0.0.1:3306)/mattermost?charset=utf8mb4,utf8&readTimeout=30s&writeTimeout=30s
+
+*PostgreSQL*
+
+.. code-block:: text
+
+   postgres://mmuser:really_secure_password@localhost:5432/mattermost?sslmode=disable&connect_timeout=10
+
+2. Create Environment File:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**NOTE:** If you are running Mattermost in an HA cluster this step must be done on **all** servers in the cluster. 
```

Comment:
```suggestion
.. note::
  If you are running Mattermost in an HA cluster this step must be done on **all** servers in the cluster. 
```

Is **HA cluster** a high availability  cluster? Would it be better to not use an abbreviation?